VaticanBAV/et/BAVet134.xml

```diff
@@ -94,11 +470,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <langUsage>
                 <language ident="en">English</language>
                 <language ident="gez">Gǝʿǝz</language>
-                
             </langUsage>
         </profileDesc>
         <revisionDesc>
             <change who="JK" when="2024-09-11">Created stub</change>
```

Comment:
https://github.com/BetaMasaheft/Documentation/issues/2700 the word "stub" must go away, thank you!